src/neuroconv/datainterfaces/ophys/brukertiff/brukertiffdatainterface.py

```diff
@@ -28,6 +29,7 @@ def get_streams(
         folder_path: DirectoryPath,
         plane_separation_type: Literal["contiguous", "disjoint"] = None,
     ) -> dict:
+        """get streams for the Bruker TIFF imaging data."""
```

Comment:
Needs Parameters section for folder_path and plane_separation_type

Reply:
Done.